x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/installation_and_upgrade.md

```diff
@@ -711,6 +711,41 @@ And the Investigation Guide tab should NOT be displayed
 
 TODO: add scenarios https://github.com/elastic/kibana/issues/166215
 
```

Comment:
I think you confused where you located these two scenarios. They shouldn't be in `### Rule upgrade workflow: filtering, sorting, pagination` but in `### Rule upgrade workflow: base cases`, in a similar way to how, when you added the scenario `#### **Scenario: User can preview a rule before installing**` you added them within `### Rule installation workflow: base cases`

Reply:
Oops, the merge of main into this branch didn't go as planned and some parts ended up duplicated and moved. Looks like I didn't remove all the wrong copies 🤦 These were originally under "base cases", indeed. Thanks for catching!